src/nrepl/connectSequence.ts

```diff
@@ -335,24 +345,62 @@ function getDefaultCljsType(cljsType: string): CljsTypeConfig {
   return defaultCljsTypes[cljsType];
 }
 
+function getDefaultSequence(
+  sequences: ReplConnectSequence[],
+  saveAsPath: string
+): ReplConnectSequence | undefined {
+  const defaultProject = getConfig().jackInProjectType;
```

Comment:
Considering my other comments, maybe we should name this and similarly named variables added in this PR to something like `userSpecifiedProjectType`.